### ISSUE TITLE
Questions for the details of summarization experiment

### ISSUE THREAD

**Issue by Alex-Ruan** [adf1178/PT4Code#9]
Hi, thank you for your work, @adf1178  I have some questions about the experiments of summarization.

1. How do you set the epochs for training, which is 20 for summarization? as we know, overfitting may occur, as for different languages, some may overfit, and others may underfit, it seems inappropriate to set them all the same.
2. For performance in low-resource scenarios, as mentioned above, overfitting occurs sometimes, how do you set the epochs for train data size 100, 200, 300, 500, and 1000 in summarization for each language? is it all the same? 20 for each?
3. For different Lengths of Prefix Soft Prompts, how do you set the epochs? so it's 20 epochs for all, or is it different for different lengths?
4. May I ask do you researchers fix the batch size and the seed for all experiments? it seems that the seed is always 42 and the batch size for training and validation is 64. As the batch size is also important for SoftTemplate (in summarization), the result changes when batch size changes when the seed is fixed (in my demo).
